x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/collapsed_item_actions.tsx

```diff
@@ -209,6 +228,18 @@ export const CollapsedItemActions: React.FunctionComponent<ComponentOpts> = ({
                 { defaultMessage: 'Disable' }
               ),
         },
+        {
+          disabled: !item.isEditable || item.consumer === AlertConsumers.SIEM,
+          'data-test-subj': 'cloneRule',
+          onClick: async () => {
+            setIsPopoverOpen(!isPopoverOpen);
+            onCloneRule(cloneRuleInternal.bind(this, item.id));
+          },
+          name: i18n.translate(
+            'xpack.triggersActionsUI.sections.rulesList.collapsedItemActons.duplicateRuleTitle',
+            { defaultMessage: 'Duplicate rule' }
```

Comment:
When I survey the existing terminology, "clone" is much more common in Kibana (e.g. "clone index", "clone job", "clone dashboard", "clone role", etc...
```suggestion
            { defaultMessage: 'Clone rule' }
```

Reply:
Perfect!, thank you for doing the research